specification/purview/resource-manager/Microsoft.Purview/stable/2021-12-01/purview.json

```diff
@@ -0,0 +1,2696 @@
+              }
+            ]
+          },
+          "eventHubType": {
+            "description": "The event hub type.",
+            "enum": [
+              "Notification",
+              "Hook"
+            ],
+            "type": "string",
+            "x-ms-enum": {
+              "name": "eventHubType",
+              "modelAsString": true
+            }
+          },
+          "eventStreamingState": {
+            "description": "The state of the event streaming service",
+            "default": "Enabled",
+            "enum": [
+              "Disabled",
+              "Enabled"
+            ],
+            "type": "string",
+            "x-ms-enum": {
+              "name": "eventStreamingState",
+              "modelAsString": true
+            }
+          },
+          "eventStreamingType": {
+            "description": "The event streaming service type",
+            "default": "None",
+            "enum": [
+              "None",
+              "Managed",
+              "Azure"
+            ],
+            "type": "string",
+            "x-ms-enum": {
+              "name": "eventStreamingType",
+              "modelAsString": true
+            }
+          },
+          "eventHubPartitionId": {
+            "description": "Optional partition Id for notification event hub. If not set, all partitions will be leveraged.",
+            "type": "string"
+          }
+        }
+      },
+      "Credentials": {
+        "description": "Credentials to access the event streaming service attached to the purview account.",
+        "type": "object",
+        "properties": {
+          "identityId": {
+            "description": "Identity identifier for UserAssign type.",
+            "type": "string"
+          },
+          "type": {
+            "description": "Identity Type.",
+            "enum": [
+              "None",
+              "SystemAssigned",
+              "UserAssigned"
+            ],
+            "type": "string",
+            "x-ms-enum": {
+              "name": "CredentialsType",
+              "modelAsString": true
+            }
+          }
+        }
+      },
+      "KafkaConfigurationList": {
+        "description": "Paged list of kafka configuration resources",
+        "required": [
+          "value"
+        ],
+        "type": "object",
+        "properties": {
+          "nextLink": {
+            "description": "The Url of next result page.",
+            "type": "string"
+          },
+          "value": {
+            "description": "Collection of items of type results.",
+            "uniqueItems": false,
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/KafkaConfiguration"
+            }
+          }
+        }
+      },
+      "OperationList": {
+        "description": "Paged list of operation resources",
+        "required": [
+          "value"
+        ],
+        "type": "object",
+        "properties": {
+          "count": {
```

Comment:
List GET endpoints (collection GET) must only have value and nextLink in properties.

refs: https://github.com/Azure/azure-openapi-validator/blob/main/docs/get-collection-only-has-value-nextlink.md